editoast/src/views/stdcm_search_environment.rs

```diff
@@ -169,13 +193,75 @@ async fn retrieve_latest(
     }
 }
 
+#[derive(IntoParams, Deserialize)]
+#[allow(unused)]
+struct StdcmSearchEnvIdParam {
+    /// An existing stdcm search environment ID
```

Comment:
```suggestion
    /// An STDCM search environment ID
```

implied by being a path parameter

Reply:
Will do ^^